core/Utilities/FileSystemUtilities.cs

```diff
@@ -448,11 +448,12 @@ public static IDirectoryInfo GetPackageOutputDir(CmfPackage cmfPackage, IDirecto
         /// </summary>
         /// <param name="packageFile"></param>
         /// <returns></returns>
-        public static XDocument GetManifestFromPackage(string packageFile)
+        public static XDocument GetManifestFromPackage(string packageFile, IFileSystem fileSystem = null)
         {
+            fileSystem ??= new FileSystem();
             XDocument dFManifest = null;
 
-            using (FileStream zipToOpen = new(packageFile, FileMode.Open))
+            using (var zipToOpen = fileSystem.FileStream.New(packageFile, FileMode.Open))

```

Comment:
we have the same issue in the methods below, but I think that we can do all these type changes during the refactor

Reply:
yeah, I only changed the methods I needed to be testable, but an overview of codecov here is needed